README.md

```diff
@@ -19,6 +19,24 @@ Once setup has been completed, you can start the server with
 script/server
 ```
 
+### Notes from a local install
+
+We additionally needed to take the following steps to get a reasonable local install outside of Docker including
+a working database for running the server locally:
+
+#### Database
+
+* Install Cloud Foundry from [GPaaS](https://www.cloud.service.gov.uk/) after logging in.
+We needed to open it in Finder to be able to install it (a second button appears in grey)
+* `cf install-plugin conduit`
+* `cf login -a api.london.cloud.service.gov.uk -u your@account.name`
+* `script/db-restore`
```

Comment:
I think this is the purpose of `doc/database-backup-and-restore.md` – which is missing the login step that you're describing here. Maybe we can move the useful detail from this section over there and link to it from here?

Reply:
Rewrote this extensively, probably worth a second glance.